packages/anchor-utils/src/constants.ts

```diff
@@ -4,13 +4,15 @@ export type SupportedNetwork =
   | 'eip155:5' // goerli
   | 'eip155:100' // gnosis
   | 'eip155:1337' // ganache
+  | 'eip155:11155111' // sepolia
 
 export const BLOCK_THRESHHOLDS: Record<SupportedNetwork, number> = {
```

Comment:
looks like this constant isn't actually used anywhere, let's just delete it.

Reply:
note there's a similar constant in ethereum-anchor-validator.ts that *is* used however, and still need to be updated.

Reply:
ping

Reply:
The type is used on line 9 above, no?
Can that be removed too?

Reply:
sorry, this comment is *on* line 9, so I'm not sure what you're referring to.  I'm saying the `BLOCK_THRESHHOLDS` constant in this file is defined but never used.

Also there's a `BLOCK_THRESHHOLDS` constant in `ethereum-anchor-validator.ts` that *is* used and I think still needs to be updated.